modules/gdscript/gdscript_tokenizer_buffer.cpp

```diff
@@ -33,7 +33,8 @@
 #include "core/io/compression.h"
 #include "core/io/marshalls.h"
 
-#define TOKENIZER_VERSION 100
+// `version = major * 100 + minor`.
+#define TOKENIZER_VERSION 101
```

Comment:
IMO it should only be updated if changed after the actual release. Dev snapshots are not meant for production and it is expected that things can break between one and the next.

Also, I see the version number as incremental, not as a combination of "major" and "minor". The only reason I started with `100` is to avoid potential conflicts with the 3.x version line if it needs to update the version number for any reason, so the 4.x versions will always be above (most likely).